modules/dlm/qa/with-security/src/javaRestTest/resources/ssl/README.asciidoc

```diff
@@ -0,0 +1,37 @@
+= Keystore Details
```

Comment:
This is copy and paste from another test, so that we have valid certificate material ssl settings. The ssl settings are not strictly necessary for the REST test, but nice to have. AFAIK, we don't currently have a good way to re-use the certs across multiple tests packages, so copying these from another test.